### DISCUSSION TITLE
Support for working with Durable Object in Cloudflare worker

### DISCUSSION THREAD

**Discussion by 4x4notfound** [remix-run/remix#4264] [Proposals]
Would be great for the cloudflare worker deployment to support Durable Objects API. DO's have lower latency and can act as mini applications within the backend. When CF figures out service binding this would also mean other workers could be bound to the remix backend.

**Comment by sergiodxa** [comment]
I think this doesn’t depend on Remix, this depends on wrangler or miniflare.

**Reply by barbinbrad** [reply]
Have you seen this?

https://github.com/jacob-ebey/remix-cloudflare-worker-template
